core/array.cpp

```diff
@@ -412,6 +412,13 @@ Array::Array() {
 	_p = memnew(ArrayPrivate);
 	_p->refcount.init();
 }
+
+Array Array::sizeInit(int size) {
```

Comment:
I don't see why this should be camelCase when everything else uses snake_case. This should probably be changed to `init_size` (assuming that the proposal is accepted)

Reply:
And method paramaters should start with `p_`, so it should be `Array::init_size(int p_size)`.